examples/icmp_socket.py

```diff
@@ -0,0 +1,57 @@
+import socket
+import sys
+
+from rflx.pyrflx import MessageValue, Package, PyRFLX
+
+
+class ICMPSocket:
+
+    package_icmp: Package
+
```

Comment:
I think this isn't necessary since you define `package_icmp` in `__init__`.

Reply:
removed it